docs/backend/db/analysis.md

```diff
@@ -1,66 +1,66 @@
 # DB row counts
```

Comment:
## Negative deltas
@Sigfried I did not expect this. This is only for 3/4 of the vocab tables (the 4th one, `relationship`, is new). Both counts tables had positive deltas though.

Do you think we should ask Amin or someone else in-the-know why they would think that this might be the case?

<img width="640" alt="Screen Shot 2023-08-08 at 11 42 31 PM" src="https://github.com/jhu-bids/TermHub/assets/13045020/8897b412-f365-4ad8-a9b9-ff589a8c5e34">

<img width="808" alt="Screen Shot 2023-08-08 at 11 43 04 PM" src="https://github.com/jhu-bids/TermHub/assets/13045020/d484a762-759d-47f3-8006-c4ceca403e04">